include/open62541/plugin/pubsub.h

```diff
@@ -65,7 +65,7 @@ struct UA_PubSubChannel {
 
     /* Receive messages. A regist to the message source is needed before. */
     UA_StatusCode (*receive)(UA_PubSubChannel * channel, UA_ByteString *,
-                             UA_ExtensionObject *transportSettings, UA_UInt32 timeout);
+                             UA_ExtensionObject *transportSettings, UA_UInt32 timeout, UA_UInt16 *noOfRecvMessages);
```

Comment:
I'm not a fan of this change:
There are two options this can be used.

1. Get the number of messages in the ByteString
We know after decoding if we have reached the end of the message bytestring.
If not, continue with the next message. So this is not required.

2. Wait until noOfRecvMessages have been received.
This is an antipattern. Blocking waits should be avoided.